### PR TITLE
Add howto hire someone to install spoke doc

### PR DESCRIPTION
**Pull request by mathemagica**
Add howto hire someone to install spoke doc

### CODE REVIEW

File: docs/HOWTO_HIRE_SOMEONE_TO_INSTALL_SPOKE.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Howto Hire Someone to Deploy Spoke
```
Comment (shakalee14) [inline docs/HOWTO_HIRE_SOMEONE_TO_INSTALL_SPOKE.md]: How to (space in between)

File: docs/HOWTO_HIRE_SOMEONE_TO_INSTALL_SPOKE.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+## Deciding to use Spoke
+
+You should treat all tech infrastructure choices as 'build vs buy' decisions, and make the right choice for your organization.
+
+Reasons to run Spoke:
+1. You want to run a peer-to-peer textbanking system at scale and pay only infrastructure costs
+2. You have the tech capacity to do the technical deployment work to set up Spoke and maintain your instance over time. 
+3. You want to be able to control costs and system scaling on your own terms. Perhaps you have experienced scaling problems from a textbanking vendor at an inopportune time. You can choose to control this risk by spending developer time and money on hardware running your own system.
+4. You care about the open source community, want to give back to the movement, and are interested in contributing back fixes and features.
```
Comment (shakalee14) [inline docs/HOWTO_HIRE_SOMEONE_TO_INSTALL_SPOKE.md]: "progressive movement"

File: docs/HOWTO_HIRE_SOMEONE_TO_INSTALL_SPOKE.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+# Howto Hire Someone to Deploy Spoke
+
+## Deciding to use Spoke
+
+You should treat all tech infrastructure choices as 'build vs buy' decisions, and make the right choice for your organization.
+
+Reasons to run Spoke:
+1. You want to run a peer-to-peer textbanking system at scale and pay only infrastructure costs
+2. You have the tech capacity to do the technical deployment work to set up Spoke and maintain your instance over time. 
+3. You want to be able to control costs and system scaling on your own terms. Perhaps you have experienced scaling problems from a textbanking vendor at an inopportune time. You can choose to control this risk by spending developer time and money on hardware running your own system.
```
Comment (shakalee14) [inline docs/HOWTO_HIRE_SOMEONE_TO_INSTALL_SPOKE.md]: Another reason/use case:

You don't want to sign a contract with a vendor, and may have a more short term need for peer to peer texting.

File: docs/HOWTO_HIRE_SOMEONE_TO_INSTALL_SPOKE.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+Please, please, please don't just hire your friends, or someone who looks like a tech person who happens to be sitting 100 ft from you in your coworking space when you realize you need a website. Most software contracting projects experience a 50% failure rate due to lack of skill fit and contractor capacity. Publish your job publicly and run a competitive process to find a person to ensure that you're getting the right person with the right skills and right availability for your job.
+
+
+*Decide on the terms of your job.*
+
+You may think you just need someone to set Spoke up for you, but you'll probably also evnetually want someone to maintain your instance over time, do regular upgrades, analyze the Spoke data you gather, and troubleshoot problems. Think about how much you want to spend upfront and in an ongoing retainer basis. Think about whether this work adds up to a full-time person or a contractor.
```
Comment (shakalee14) [inline docs/HOWTO_HIRE_SOMEONE_TO_INSTALL_SPOKE.md]: "eventually"

File: docs/HOWTO_HIRE_SOMEONE_TO_INSTALL_SPOKE.md (hunk 3 above)
Comment (mathemagica) [reply docs/HOWTO_HIRE_SOMEONE_TO_INSTALL_SPOKE.md]: ++

**Review by shakalee14 — APPROVED** [review]

**Pull request merged**